CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+## 5.0.0 - 2020-09-02
+
+### Changed
+
+We have replaced the use of the npm [request-promise](https://www.npmjs.com/package/request-promise) package with [axios](https://www.npmjs.com/package/axios) as the npm `request` package has been deprecated. This makes the following breaking changes.
+
+1. The `response` `object` returned by a successful API call is now of the form of an [axios response](https://www.npmjs.com/package/axios#response-schema) which has a different interface to a [request response](https://nodejs.org/api/http.html#http_class_http_incomingmessage), of particular note
+
+    * `response.body` becomes `response.data`
+    * `response.statusCode` becomes `response.status`
+
+2. The `err` `object` returned by an unsuccessful API call has a different interface, of particular note
+
+    * `err.error` becomes `err.response.data`
+
+    See the axios documentation for further details on [error handling](https://www.npmjs.com/package/axios#handling-errors).
+
+3. To configure the use of a proxy you should pass the proxy configuration as an object as per [axios](https://github.com/axios/axios) rather than a URL.
```

Comment:
```suggestion
1. The `response` `object` returned by a successful API call is now in the form of an [axios response](https://www.npmjs.com/package/axios#response-schema). This has a different interface to a [request response](https://nodejs.org/api/http.html#http_class_http_incomingmessage). Please note that:

    * `response.body` becomes `response.data`
    * `response.statusCode` becomes `response.status`

2. The `err` `object` returned by an unsuccessful API call has a different interface. Please note that `err.error` becomes `err.response.data`.

    See the axios documentation for further details on [error handling](https://www.npmjs.com/package/axios#handling-errors).

3. To configure the use of a proxy you should pass the proxy configuration as an object rather than a URL. For details, see the [axios client](https://github.com/axios/axios).
```

Reply:
I've tried to bring sentence length down a bit. I also removed the use of a single bullet.

One other thing I considered – but wasn't sure about – was asking users to take an action, for example:

>Please change:
>    * `response.body` to `response.data`
>    * `response.statusCode` to `response.status`

Reply:
I'm wondering if we change it to `For example` rather than `please note that`. 

For context, there will be likely be several differences between the old interface and the new interface. I have listed the examples that I assume developers would most likely run in to and would hopefully save them the initial trouble of looking up but don't want them to get the opinion that the examples are the only ones. Maybe `For example` would do that best?

Reply:
I didn't suggest `For example` before because I assumed these were specific things that users need to act on.

If you're comfortable with just calling them out, I definitely think `For example` reads better.

Reply:
>For example:
>    * `response.body` becomes `response.data`
> ...

And

>For example, `err.error` becomes `err.response.data`.